functions/src/expressRestApp/index.ts

```diff
@@ -0,0 +1,20 @@
+import dotenv from "dotenv";
+import express from "express";
+import * as functions from "firebase-functions";
+import stationsRoutes from "./routes/route.service";
+import handleException from "./middleware/uncaughtErrorsHandler";
+import errorHandler from "./middleware/unhandledErrorsHandler";
+
+dotenv.config();
+
+const app = express();
+
+app.use(express.json());
+
+handleException();
+
```

Comment:
we may add handler for base url here to notify that our service is working:
`app.use('/', (req: Request, res: Response, next: NextFunction ): void => {
  if (req.originalUrl === '/') {
    res.send('Service is running!');
    return;
  }
  next();
});`